### ISSUE TITLE
Need to micromanage brightness on resume and VT switch

### ISSUE THREAD

**Issue by Mikroprocessor** [dnschneid/crouton#407]
Hi!

I have Ubuntu 13.04 on my Acer C7 Chromebook.
Every time i lay down the screen(as a lid on a jar, but not screwing down it) and wait until the laptop has gone to sleep-mode. Then i open the screen again the screen is black. If then i plug in (or plug out) the charger plug(not it plug who fits in the wall, the plug who fits in the round hole on the laptop), then the screen wakes up and i can you the laptop again.

I don't know why it's on that way, but im sure that is a bug. Right?

Med vänliga hälsningar - Best regards
Sweden.

**Comment by ccaapton** [comment]
@dnschneid I'm also experiencing ChromeOS misbehaving, but not crouton using awesomewm. Is there any workaround to avoid reboot? Thx!

**Comment by midnightradio** [comment]
I'm having the same issue on my pixel. It reboots on opening the lid occasionally. At first I thought it's only happening for running xfce and started to keep doing some jobs with just entering chroot without starting the desktop manager. But still it happened even if I didn't run xfce. So I guess it's not a matter of xscreensaver / power manager or desktop environment. Also it keeps happening even when I changed focus to a tab other than the tab for chroot before closing the lid. 

I can close the lid only after saving all file I was working on. :-(

**Comment by dnschneid** [comment]
@midnightradio could you please file a separate issue for that?

**Comment by dnschneid** [comment]
Are people still hitting this bug?
